tests/semantic_analyser.cpp

```diff
@@ -219,7 +219,7 @@ TEST_F(SemanticAnalyserTest, builtin_variables)
   test("kprobe:f { func }");
   test("uprobe:/bin/sh:f { func }");
   test("kprobe:f { probe }");
-  test("tracepoint:a:b { args }");
+  test("tracepoint:sched:sched_one { args }", Mock{ *get_mock_bpftrace() });
```

Comment:
JC, why does this have to be wrapped in a new `Mock`?

Reply:
It doesn't, I just missed that the `test` method already creates a `MockBPFTrace` if it's not passed. Dropped.